installer/templates/new/web/web.ex

```diff
@@ -48,9 +48,11 @@ defmodule <%= application_module %>.Web do
     end
   end
 
-  def view do
+  def view(opts \\ []) do
```

Comment:
Ok, now I see your vision. Just make load path configurable but not out of the box.